Formula/autoenv.rb

```diff
@@ -16,4 +16,11 @@ def caveats; <<-EOS.undent
       source #{opt_prefix}/activate.sh
     EOS
   end
+
+  test do
+    (testpath/"test").mkdir
+    (testpath/"test/.env").write "echo it works"
+    testcmd = "yes | bash -c '. #{opt_prefix}/activate.sh; autoenv_cd test'"
```

Comment:
I think generally our use of a mixture of pipe_output and shell_output depending on circumstances when `|` could be used instead is kind of weird but :man_shrugging: